repo/packages/O/openldap/2/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "packagingVersion": "2.0",
```

Comment:
can I simply change this or does it have any other implications? TBH I was reusing from other existing packages so if there's anything I can read on what this implies I'll check it out.